src/ultra/test.clj

```diff
@@ -43,6 +44,19 @@
         (diff/print-expected actual expected))
       (logic/maybe-print-values event)))
 
+(defmethod report :error [m]
```

Comment:
Also I notice you're individually accessing keys of `m` later on; you should copy the destructuring seen in `:fail` above for consistency.